debian/control

```diff
@@ -92,7 +92,7 @@ Build-Depends: cmake (>= 3.5),
 # Make-Check   uuid-dev,
                uuid-runtime,
                valgrind,
-               virtualenv | python-virtualenv,
+               virtualenv | python3-virtualenv,
```

Comment:
`virtualenv` would suffice. if we are targeting a py3 distro. then, we can assume `virtualenv` depends on `python3-virtualenv` there.

Reply:
when you say _"py3 distro"_ do you mean a Python3-exclusive one? In Bionic Python3 comes in by default but you can install Python 2.7 which creates a confusing (in-transition) situation. I would rather not assume that `virtualenv` is going to depend on `python3-virtualenv`

Reply:
i don't think there is a "python3-exclusive" distro. by "py3" distro, i mean a distro which is taking python3 a first-class citizen: which means all its python (related) packages are python3 ready.